src/cluster.rs

```diff
@@ -723,6 +723,13 @@ impl RoutingInfo {
                     get_arg(&args, 3).and_then(RoutingInfo::for_key)
                 }
             }
+            b"XGROUP" | b"XINFO" => get_arg(&args, 2).and_then(RoutingInfo::for_key),
+            b"XREAD" | b"XREADGROUP" => {
+                let streams_position = args
+                    .iter()
+                    .position(|a| *a == Value::Data(b"STREAMS".to_vec()))?;
```

Comment:
Mini-optimisation: we can pull out the `Value::Data(b"STREAMS".to_vec())` before the whole iteration, therefore saving us the frequent re-creation (and thus re-allocation)

Reply:
I missed that somehow, thanks.

It should be ok now?